modular_exponential/modular_exponential.py

```diff
@@ -0,0 +1,33 @@
+"""
+Modular Exponential
+"""
+
+def mod_exponent(base, power, mod):
+    """
```

Comment:
@yashLadha Sir actually under Multi-Line docstring it is mentioned `Multi-line docstrings consist of a summary line just like a one-line docstring, followed by a blank line, followed by a more elaborate description.` .